doc/source/user_guide/hpc/troubleshooting.rst

```diff
@@ -19,44 +50,91 @@ Python virtual environment is not accessible
 --------------------------------------------
 If there is an error while testing the Python installation, it might mean 
 that the Python environment is not accessible to the compute nodes.
-For example, in the following output, PyMAPDL could not be found, meaning that the script
-is not using the virtual environment (``/home/user/.venv``):
+For example, given the following *bash* script `test.sh`:
+
+.. code-block:: bash
+
+   source /home/user/.venv/bin/activate
+   python -c "from ansys.mapdl import core as pymapdl; pymapdl.report()"
+
+The following output is shown after running in the terminal:
 
 .. code-block:: console
 
     user@machine:~$ srun test.sh
+
     Testing Python!
     Traceback (most recent call last):
     File "<string>", line 1, in <module>
     ImportError: No module named ansys.mapdl
 
-This could be for a number of reasons. One of them is that the system Python distribution
-used to create the virtual environment is not accessible from the compute nodes
-due to one of these reasons:
+As the output shows, PyMAPDL could not be found, meaning that either:
+
+* The virtual environment does not have PyMAPDL installed.
+  See :ref:`ref_install_pymapdl_on_hpc`.
+
+* Or the script did not properly activate the virtual environment
+  (``/home/user/.venv``).
 
-- The virtual environment has been created in a
-  directory that is not accessible from the nodes.
-- The virtual environment has been created from a Python
-  executable that is not available to the compute nodes.
-  Hence, the virtual environment is not activated. For
-  example, you might be creating the virtual environment
-  using Python 3.10, but only Python 3.8 is available
-  from the compute nodes.
+For the second reason, there could be a number of reasons.
+One of them is that the system Python distribution used to create
+the virtual environment is not accessible from the compute nodes
+due to one of these reasons:
```

Comment:
```suggestion
The second problem can occur due to a number of reasons.
One of them is that the system Python distribution used to create
the virtual environment is not accessible from the compute nodes
because of one of these situations:
```
Tried to rewrite to avoid the use of so many "reasons"!

Reply:
Trueee.. sounds so much better now!